src/types/multiModel.js

```diff
@@ -15,14 +13,12 @@ class MultiModel extends BaseType {
         this.options = options || {};
         this.map = this.options.map;
         this.Model = Model;
-        this.requestCache = new RequestCache();
         this._listeners = {};
-        this._models = {};
     }
 
     _instantiateModel (id: ?string): Object {
         const { Model } = this;
-        const model = new Model(this._parent, this._parent, id, this.options.modelOptions);
+        const model = new Model(this._parent, id, this.options.modelOptions);
```

Comment:
Right, but that Event still needs to *belong* to the exposition. The idea of hasMany is that it is an array of pointers to models in an OwnsMany. If you cant hit `/scopes/:id/models/:id` then the model isn't scoped correctly. If you set the scope of a Performer to an event and then called fetch on it, you would be hitting a URL that looks like this: `/expositions/1/events/11/performers/8` and then you would get a 404.